### ISSUE TITLE
Debian maintainer <- Volunteer needed, help!

### ISSUE THREAD

**Issue by walles** [walles/moor#137]
`moar` should be part of Debian.

Kindly contact the Debian Go maintainers about this: https://go-team.pages.debian.net/

As a positive side effect, this would also make `moar` part of Ubuntu and other Debian derivatives.

If you decide to maintain `moar` for Debian, please close [the associated Request For Packaging](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=944035).

# Resources
* https://go-team.pages.debian.net/
* https://wiki.debian.org/Packaging
* https://packaging.ubuntu.com/html/packaging-new-software.html

**Comment by Jelloeater** [comment]
@walles I wrote a nice workflow for releasing deb / yum packages, if you would be interested?
https://github.com/Jelloeater/stampy/blob/main/.github/workflows/release.yml
LMK?
I'm using packagecloud.io and it works well.

**Comment by walles** [comment]
Thanks @Jelloeater!

This got me thinking about what I'm really after here and I came to two conclusions:
* I want not just a Debian *package*, but a Debian *maintainer* to maintain a `moar` package in Debian
* Until then, Debian users can just as well use Homebrew or `go install` for their `moar` needs

I don't think packagecloud.io would add any value for `moar` users, since AFAIU that's just another third party repository just like Homebrew / `go install`, and `moar` is already maintained there.

But maybe I'm just wrong there?

With a Debian maintainer though, `moar` would magically show up for Ubuntu as well, and other Debian derivatives, like my `ps` replacement:
* `px` is maintained for Debian: https://packages.debian.org/bookworm/px
* This makes `px` magically show up for Ubuntu as well: https://packages.ubuntu.com/noble/px

I have attempted to clarify the title and description of this ticket, and [boosted Homebrew as an install option on Linux](https://github.com/walles/moar/commit/d5f1cbd43d83c9e40347f6ef95665dbd72a65ac0).

**Comment by Jelloeater** [comment]
@walles 

Yeah, that would be great for sure. I've just seen it can be hard for smaller applications to get accepted into the main line distros, so I don't bother for my little personal stuff, but for YOUR app, it's WAY more popular and would def be great to just let folks be able to `apt install moar` for sure!

You might want to include toss in https://github.com/marwanhawari/stew as a option for installing? I use it for all my binary CLI app.

It seems like ALOT of work to get it included tho :-(

https://unix.stackexchange.com/questions/620672/how-can-i-publish-a-deb-package

![image](https://github.com/user-attachments/assets/d2e00113-5e75-4c27-97eb-3ad9d3c288ca)

Am I reading this right?

**Comment by walles** [comment]
Maybe try to contact these people? I think I did that using their mailing list, but I can't seem to find any reference to that any more:

https://go-team.pages.debian.net/

The RFP has been open for a long time:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=944035

**Comment by walles** [comment]
I just wrote an e-mail to `debian-go at lists.debian.org`, let's see what happens:

https://lists.debian.org/debian-go/2025/08/msg00013.html

**Comment by walles** [comment]
`moor` is waiting in the NEW queue to be included in Debian Sid / Unstable / Forky: https://ftp-master.debian.org/new.html

**Comment by walles** [comment]
Done: https://packages.debian.org/sid/moor
